services/meta/parameters.md

```diff
@@ -0,0 +1,149 @@
+# Parameters Service
+
+Parameters service allows configuration of a device.
+
+## Service name
+
+`parameters`
```

Comment:
in general this doc is hard to read due to the "flat" structure - all elements can put stuff on the same lvl (`min`/`max` and `options`), and we end up with optional stuff everywhere.